### PR TITLE
chore(deps): bump sanity to v2.29.3 and adjust code after the upgrade

### PR DESCRIPTION
**Pull request by artzag**
## Scope

bump sanity to v2.29.3 and adjust code after the upgrade.

## Checklist
I have:
- [x] used [conventional commit messages](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] committed with git-hooks active (ran npm install at least once locally before committing)
- [x] considered if this is a breaking change
- [x] tested the changes using npm link / yarn link
- [x] triggered nrkno-sanity-libs-releaser for this branch and checked the status

The branch-build must be manually triggered by the nrk.no team using nrkno-sanity-libs-releaser.

After merging, new version(s) should be published to npm by by triggering
nrkno-sanity-libs-releaser for the master branch.

These manual steps are a safeguard to prevent accidental releases and version changes.

### CODE REVIEW

**Comment by rosvoll** [comment]
Ser jo greit ut for meg, jeg skjønner ikke 100% hvordan dette henger sammen. Men nå bruker vi v 2.29.3 her, mens vi bruker 2.29.2 i nrkno-sanity?

**Comment by pingrid** [comment]
Gjør du en test med npm link i nrkno-sanity-repo og testkjøring av bygget på jenkins? 😊 Og gir beskjed når du har gjort det?

**Review by AndersCan — APPROVED** [review]
det de andre sa (npm link) og forklaring på hvorfor vi trenger `__experimental_search`

ellers 👍

File: packages/sanity-plugin-nrkno-spellcheck/src/lib/ui/components/SpellcheckButtonTooltips.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React, { PropsWithChildren, useCallback, useContext } from 'react';
 import { Box, Button, Card, Flex, Label, Popover, Stack, Tooltip } from '@sanity/ui';
 import { SpellcheckDispatch } from './SpellcheckContext';
 import { SpellcheckProgressbar } from './SpellcheckProgressbar';
-import { TooltipProps } from '@sanity/ui/lib/src/primitives/tooltip/tooltip';
+import { TooltipProps } from '@sanity/ui/lib/dts/primitives/tooltip/tooltip';
```
Comment (ajaco) [inline packages/sanity-plugin-nrkno-spellcheck/src/lib/ui/components/SpellcheckButtonTooltips.tsx:5]: Kunne vi brukt 
```suggestion
import type { TooltipProps } from '@sanity/ui/lib/dts/primitives/tooltip/tooltip';
```

Her?

**Review by pingrid — APPROVED** [review]
Bra forresten 👏  Gjenstår fortsatt testingen ?

File: packages/sanity-plugin-nrkno-spellcheck/src/lib/core/document/document-spellcheck-values.test.ts [hunk 2]
```diff
@@ -251,6 +251,7 @@ describe('spellcheck-values', () => {
               },
             },
           ],
+          __experimental_search: [],
```
[the remark this answers is not in the document]
Comment (artzag) [reply packages/sanity-plugin-nrkno-spellcheck/src/lib/core/document/document-spellcheck-values.test.ts:254]: Feltet er required i `ObjectSchemaType` fra Sanity.

**Pull request merged**